src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -36,9 +36,8 @@ class HoldRequest extends React.Component {
   }
 
   render() {
-    const item = this.props.item;
     const searchKeywords = this.props.searchKeywords;
-    const record = this.props.item;
+    const record = this.props.bib;
     const title = record.title[0];
```

Comment:
We need some fallbacks here. I tried typed random IDs (as the API is still not working :-/) and the page will be broken.